### DISCUSSION TITLE
pressing the download icon in the app deletes image from immich

### DISCUSSION THREAD

**Discussion by nordic-style** [immich-app/immich#9761] [Q&A] [answered]
### The bug

Clicking on "download icon" in the picture view will delete the picture from the app and the server.

![image still there](https://github.com/immich-app/immich/assets/11313330/e48a6a4a-d28f-4d48-bbd7-87af3a925e3c)
![download](https://github.com/immich-app/immich/assets/11313330/16112270-84cb-414f-ad50-53f133bd3eeb)
![deleted](https://github.com/immich-app/immich/assets/11313330/0e8d96f7-91e7-4bfe-9388-52d24b57b39a)

### The OS that Immich Server is running on

ubuntu 220.04

### Version of Immich Server

v1.105.1

### Version of Immich Mobile App

v1.105.0 build.139

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
not required
```

### Your .env content

```Shell
not required
```

### Reproduction steps

```bash
1. go to details of a picture
2. press download
3. picture has been deleted from the server
```

**Comment by mmomjian** [comment, marked as answer]
The icon you have circled means "[archive](https://pictogrammers.com/library/mdi/icon/archive-arrow-down-outline/)". Please check the Archive tab on the app and/or server.

The download icon is the cloud in the top right.

**Reply by nordic-style** [reply]
ok - thank you. sorry for my ticket. it wasn't that obvious to me.
